client/src/components/DataFiles/DataFilesModals/DataFilesCopyModal.js

```diff
@@ -63,15 +63,10 @@ const DataFilesCopyModal = React.memo(() => {
     });
 
   const onOpened = () => {
-    const systemParams = {
-      api: 'tapis',
-      scheme: 'private',
-      system: systems[0].system
```

Comment:
Hmm, I think we want the modal to open up into the top level of the system list browser when copying. We disable moving between systems, so that's why it's different there. That said, I'm not sure I can replicate the original issue. Can you?

Reply:
Yes, I can reproduce the issue. Have you tried pointing your community data at a2cps.storage.community?
![Screen Shot 2021-12-20 at 10 53 21 AM](https://user-images.githubusercontent.com/7462330/146803946-f2504fe0-a8af-4a73-a477-93aecde799e5.png)

Reply:
You're right, this fixes the error, but it does open up the copy modal into the current directory of the source file. Can we make it so that it still opens the top level directory of the target system like before?

Reply:
@fnets  e.g. "Copy" opens the modal directly to the folder containing the target file:
<img width="1118" alt="Screen Shot 2022-01-05 at 2 14 44 PM" src="https://user-images.githubusercontent.com/20326896/148282986-30eacc1e-cb49-4a7f-8c0f-63cf986ba704.png">

Reply:
Think I got it